vscode-trace-extension/package.json

```diff
@@ -30,6 +30,11 @@
         "command": "traces.openTraceFile",
         "title": "Open with Trace Viewer",
         "icon": "assets/media/dep.svg"
+      },
+      {
+        "command": "openedTraces.openTraceFile",
```

Comment:
It's actually a trace file. With the open dialog only a folder can be selected. Then open procedure scans for CTF traces recursively. So, this should be renamed. The open dialog only supports either folder selection or file selection, and not both.

The other, existing command that can be invoked from the file explorer and context-sensitive menu is a bit different. It allows also for selecting a single file trace (not CTF traces).